shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/AbstractShenyuPlugin.java

```diff
@@ -128,13 +128,13 @@ private Boolean filterRule(final RuleData ruleData, final ServerWebExchange exch
     }
 
     private void selectorLog(final SelectorData selectorData, final String pluginName) {
-        if (selectorData.getLogged()) {
+        if (Boolean.TRUE.equals(selectorData.getLogged())) {
```

Comment:
why modify this？

Reply:
`Boolean logged` maybe is null , if null ,thow NullPointException